Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -402,6 +402,39 @@ public HttpResponseMessage PatchAgreementConcludedAt(int id, [FromBody] SingleVa
                 .Match(_ => Ok(), FromOperationError);
         }
 
+        [HttpPatch]
+        [Route("{id}/oversight-option")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage PatchOversightOption(int id, [FromBody] SingleValueDTO<YearMonthIntervalOption> oversightInterval)
+        {
+            if (oversightInterval == null)
+                return BadRequest("YearMonthIntervalOption must provided");
```

Comment:
det hedder parameteren ikke - brug nameof of string interpolation. Så følges evt. navngivninger ad